data/monitoring/prom-rule.yaml

```diff
@@ -22,6 +22,7 @@ spec:
           expr: sum(kubevirt_cnao_nmstate_deployed or vector(0)) > 0 and sum(kubevirt_nmstate_operator_deployments or vector(0)) == 0
           for: 5m
           labels:
+            priority: high
```

Comment:
@rhrazdil Please move the Priority label below the severity.

Reply:
IMHO semantically, the required fields should go before the optional ones,
using that way of thinking, `severity` that is  required label (and more important) will go before the optional `priority`.
I have nothing against going with alphabetical order if it's the common practice for labels, so that is also good option, don't know what is best.

Reply:
@rhrazdil I agree with Kobi.

Reply:
Okay, I take it. [Updated](https://github.com/kubevirt/cluster-network-addons-operator/compare/46cadb76d39bfc90514006a71397e8553100edef..faf97e0175d2f1fa3bd071b60a3f44fcccd668ca)